ui/src/app/shared/components/duration-panel.tsx

```diff
@@ -17,3 +18,21 @@ export const DurationPanel = (props: {phase: NodePhase; duration: number; estima
     }
     return <>{formatDuration(props.duration)}</>;
 };
+
+export const DurationFromNow = ({getDate, frequency = 1000}: {getDate: () => void; frequency?: number}) => {
+    const [now, setNow] = React.useState(moment());
+    const [date, setDate] = React.useState(getDate);
+    React.useEffect(() => {
+        const interval = setInterval(() => {
+            setNow(moment());
+            setDate(getDate);
+        }, frequency);
+        return () => {
+            clearInterval(interval);
+        };
+    }, []);
+
+    const duration = moment.duration(moment(date).diff(now)).asSeconds();
+
+    return <>{formatDuration(duration)}</>;
```

Comment:
could you use <TIcker/> for this?

Reply:
Sorry, I don't quite understand the comment,
would you mind providing me more detail

Reply:
There is a component called <Ticker/> that I think does this

Reply:
There's `<Moment duration={now} date={date} />`.
I thought we want to keep the formatting the same, thus using `formatDuration`